Initium-Odp/war/odp/javascript/script.js

```diff
@@ -1009,21 +1008,24 @@ function reloadPagePopup(quietly)
 {
 	if (currentPopupStackIndex==0)
 		return;
-	
+
 	var pagePopupId = "page-popup"+currentPopupStackIndex;
 	var content = $("#"+pagePopupId+"-content");
-	
+	var iframe = content.find("#"+pagePopupId+"-iframe");
+	if (iframe!=null)
+		content = iframe;
+
 	var url = content.attr("src");
+
+	if (quietly)
```

Comment:
I'm on my phone so this is a little bit harder but it appears that this line is using the loading icon when quietly is set to true. Quietly means the reload happens without disrupting the existing content. It's meant to allow the user to continue using the page while the load is taking place in the back ground.